test/unit/app/controllers/network/stubs.js

```diff
@@ -1,5 +1,4 @@
 import {
-  TRANSACTION_CATEGORIES,
   TRANSACTION_STATUSES,
   TRANSACTION_TYPES,
```

Comment:
There are two references to the now-deleted `STANDARD` type in this file

Reply:
Fixed up, thanks for finding!